sdk/python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pulumi_esc_sdk"
-version = "0.1.1-dev.0"
+version = ""
```

Comment:
Do we want a version here?

Reply:
No, because it populated at build/publish time.